site/content/en/docs/Installation/_index.md

```diff
@@ -45,17 +43,17 @@ Each version of Agones supports a specific version of Kubernetes. When a new ver
 
 The following table lists recent Agones versions and their corresponding required Kubernetes versions:
 
-| Agones version | Kubernetes version |
-| -------------- | ------------------ |
-| 1.29           | 1.24               |
-| 1.28           | 1.23               |
-| 1.27           | 1.23               |
-| 1.26           | 1.23               |
-| 1.25           | 1.22               |
-| 1.24           | 1.22               |
-| 1.23           | 1.22               |
-| 1.22           | 1.21               |
-| 1.21           | 1.21               |
+| Agones version | Kubernetes version   |
+| -------------- | ------------------   |
+| 1.29           | {{% k8s-version %}}  |
```

Comment:
This needs to be behind a release guard so that it doesn't get published when your PR merges. It should be added in as part of the 1.30 release (since we aren't retroactively adding support for new versions to 1.29).

Reply:
I think it should be fine.
After my PR merges, {{% k8s-version %}} is still 1.24 for production environment. For 1.30 release, I explictly added the release step to modify this part:
"for the current version, replace `{{% k8s-version %}}` with hardcoded Kubernetes versions supported by the current version. And add a row for the Agones release version with `{{% k8s-version %}}` as its supported Kubernetes versions"

Reply:
That makes sense.